### PR TITLE
Add project plan, git logs

### PR DESCRIPTION
**Pull request by ishaankolluri**
Added brief bit on our process and dev, added git logs as per the convention

### CODE REVIEW

File: tex/plan.tex [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+Our goals were outlined by weekly meetings. We regularly met with Jacob Graff, our advisor throughout the development of Extend.
+Jacob served as a sounding board whenever Extend's fundamental design philosophy was debated, and as a guide as we determined whether we were on track. We used any leftover time on those days to set goals for the upcoming week and pair program if time permitted.
+\newline \newline
+Our team also met weekly on Fridays to further discuss the progression of Extend. In the first half of the semester, the discussions were primarily philosophical, as decisions had to be made about the language grammar and behavior of certain Extend artifacts prior to development. In the second half, time was devoted to ironing out the development timeline, discussing bugs, and making compiler implementation decisions.
+
+\section{Development Workflow}
+
+  \subsection{Github \& Travis CI}
+  Our development and documentation were all done entirely through version control to maximize independent productivity.
+  New features were introduced to the master branch through pull requests, and the team collectively had to approve the code in order to merge each feature permanently.
```
Comment (Neitsch) [inline tex/plan.tex]: Factually the team did not have to collectively approve. It is more along the lines of review to ensure code quality.

File: tex/plan.tex [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  \medskip \noindent
+  An important aspect of development for us was continuous integration. We used Travis CI to trigger project builds on each pull request, which kept us informed regarding unexpected hiccups that sometimes arose during development. Travis CI enforced accountability to ensure each feature was properly implemented, rebased, and shipped.
```
Comment (Neitsch) [inline tex/plan.tex]: Also ensured that nothing broke. Don't think rebased makes particular sense here

File: tex/plan.tex [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+\section{Team Member Responsibilities}
+
+\begin{tabular}{ | l | l | l |}\hline
+  Team Member  & Responsibilities      & GitHub Profile\\ \hline
+  Jared Samet & design philosophy, semantic transformations, code generation  & oracleofnj\\
+  Nigel Schuster & development protocol, code generation, scripting  & Neitsch\\
+  Ishaan Kolluri & initial LRM, Final Report, regression tests, stdlib functions, scripting & ishaankolluri\\
+  Kevin Ye & initial scanner, regression tests, stdlib functions & kevinye1\\ \hline
```
Comment (Neitsch) [inline tex/plan.tex]: Make github names to links

**Review by Neitsch — COMMENTED** [review]
Sorry for my short comments. Pretty tired right now.

**Review by Neitsch — APPROVED** [review]
Kinda crazy that we print 75 pages of logs 😨 
Will be more than 80 by the end.

**Comment by ishaankolluri** [comment]
Yeah -- that plus the code listing is a ton of pages.

**Comment by Neitsch** [comment]
It just occured to me, that it might be valuable to talk a little more about our PR process, since we are really utilizing this extensively. Any changeset to master must (1) pass CI (2) be approved by another team member (3) up to date with master.

**Comment by ishaankolluri** [comment]
That's a good point. I also have to change the git logs in this section to be a little more concise. Will put out a new PR on this then

**Comment by Neitsch** [comment]
I'd suggest using 
```
git log --pretty=format:"%h %cI %cn: %s"
```
Optionally add graph

**Pull request merged**